test/e2e-go/restAPI/restClient_test.go

```diff
@@ -1392,4 +1403,32 @@ end:
 	a.NoError(err)
 	a.Empty(resp)
 	a.Zero(createdBoxCount)
+
+	// Get Box value from box name
+	encodeInt := func(n uint64) []byte {
+		ibytes := make([]byte, 8)
+		binary.BigEndian.PutUint64(ibytes, n)
+		return ibytes
+	}
+
+	boxTests := [][]interface{}{
+		{[]byte("foo"), "str:foo", []byte("bar12")},
+		{encodeInt(12321), "int:12321", []byte{0, 1, 246, 247, 2}},
+		{[]byte{0, 248, 255, 0}, "b64:APj/AA==", []byte("lux56")},
```

Comment:
This is a good test - I'm glad it contains '/'. Just to be on the safe side, please do one that contains a space, and a null byte.